test/include/scenes.hpp

```diff
@@ -47,3 +47,26 @@ static inline void setupBoxesAlongAxes(rgl_scene_t scene)
 		EXPECT_RGL_SUCCESS(rgl_entity_set_pose(zs[i], &zTf));
 	}
 }
+
+static inline void setupThreeBoxScene(rgl_scene_t scene)
+{
+        std::vector<rgl_entity_t> xs, ys, zs;
+
+        rgl_entity_t boxA, boxB, boxC;
+
+        boxA = makeEntity(makeCubeMesh(), scene);
+        boxB = makeEntity(makeCubeMesh(), scene);
+        boxC = makeEntity(makeCubeMesh(), scene);
+
+        rgl_mat3x4f boxATf = Mat3x4f::TRS({ 6, -5, 0 }, { 0, 0, 0, }, { 1, 1, 1 }).toRGL();
+        rgl_mat3x4f boxBTf = Mat3x4f::TRS({ 6,  0, 0 }, { 0, 0, 0, }, { 1, 1, 1 }).toRGL();
+        rgl_mat3x4f boxCTf = Mat3x4f::TRS({ 6,  5, 0 }, { 0, 0, 0, }, { 1, 1, 1 }).toRGL();
+
+        EXPECT_RGL_SUCCESS(rgl_entity_set_pose(boxA, &boxATf));
+        EXPECT_RGL_SUCCESS(rgl_entity_set_pose(boxB, &boxBTf));
+        EXPECT_RGL_SUCCESS(rgl_entity_set_pose(boxC, &boxCTf));
+
+        EXPECT_RGL_SUCCESS(rgl_entity_set_id(boxA, 1));
+        EXPECT_RGL_SUCCESS(rgl_entity_set_id(boxB, 2));
+        EXPECT_RGL_SUCCESS(rgl_entity_set_id(boxC, 3));
+}
```

Comment:
After giving it a thought - I don't like the existence of this function. It hardcodes three, positions and sets instance IDs. Practically no reusability. I'd prefer to have a function taking needed args (with optional entity id) and calling it three times.

Reply:
I will make a compromise